### DISCUSSION TITLE
What's the meaning of "forward proxy"?

### DISCUSSION THREAD

**Discussion by CarthagoDelendaEst** [shadowsocks/shadowsocks-windows#3362] [Q&A]
I'm a little out of the loop, what does this setting mean? when it should be used?
![изображение](https://user-images.githubusercontent.com/25225971/201482051-ece89c2e-2abb-4f9a-bc96-ea367390fa73.png)

**Comment by chenshaoju** [comment]
If you need to use a proxy to access the Internet(like a Socks5 or HTTP Proxy), you can set it up here.
